Formula/lucid.rb

```diff
@@ -0,0 +1,17 @@
+class Lucid < Formula
+  desc "Command-line tool for Lucid"
+  homepage "https://github.com/scribd/Lucid"
+  url "https://github.com/scribd/Lucid/archive/1.0.0.tar.gz"
+  sha256 "298b3a42047930956bb1bbab9a22fc57433b10842089af31d6df86e459125234"
+  license "MIT"
+
+  depends_on xcode: ["11.4", :build]
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/lucid", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula